core/src/subgraph/instance_manager.rs

```diff
@@ -706,6 +709,23 @@ where
 
             match res {
                 Ok(needs_restart) => {
+                    // Once synced, no need to try to update the status again.
+                    if !synced &&
+                        // deployment.head == chain.head
```

Comment:
I'd remove this comment, it looks like commented out code

Reply:
Done!